packages/@ember/-internals/glimmer/tests/integration/syntax/each-in-test.js

```diff
@@ -672,6 +672,44 @@ moduleFor(
       </ul>
     `);
     }
+
+    [`@test it supports native ES6 Sets`]() {
```

Comment:
Hmm, why use `{{#each-in` with a `Set`? I would have expected to use `{{#each` 🤔

Reply:
Agreed, but this behavior did exist beforehand. We could throw an error instead potentially, not sure if it was considered public API.

Reply:
Throwing seems better to me, also I assume this is just something you noticed in passing (since there are no linked issues)?

Reply:
Yeah. Ok, sounds good 👍